### DISCUSSION TITLE
Batch delete

### DISCUSSION THREAD

**Discussion by yordan-lyutskanov** [ory/kratos#3425] [Q&A]
As per the docs the https://www.ory.sh/docs/kratos/reference/api#tag/identity/operation/batchPatchIdentities endpoint should be able to create/delete multiple identities. However it is not documented how to use the delete functionality of this endpoint, so I had to check the code. While checking I couldn't find any support for batch delete. Is this even working or is it an error in the docs?

**Comment by vinckr** [comment]
Hello @yordan-lyutskanov 
What functionality are you looking for? Deleting thousands of users is not something people usually do, so you can currently delete the users using [deleteIdentity](https://www.ory.sh/docs/reference/api#tag/identity/operation/deleteIdentity).
